spec/api/process_spec.rb

```diff
@@ -1328,39 +1328,36 @@
     end
   end
 
-  it "can create and retrieve silenced registry entry with a subscription containing a colon" do
-    api_test do
-      options = {
-        :body => {
-          :subscription => "client:test",
-          :check => "test",
-          :expire => 3600,
-          :reason => "testing",
-          :creator => "rspec",
-          :expire_on_resolve => true
+  [
+    "client:my-test-client",
+    "roundrobin:load_balancer",
+    "roundrobin:foo_bar-baz",
+    "load-balancer",
+    "load_balancer",
+    "loadbalancer"
+  ].each do |sub|
```

Comment:
The only criticism that I have for you, why not `subscription`? We've avoided the use of short and single character variables when possible.